commands.json

```diff
@@ -0,0 +1,144 @@
+                "optional": true,
+                "multiple": true
+            },
+            {
+                "name": "arg",
+                "type": "string",
+                "optional": true,
+                "multiple": true
+            }
+        ]
+    },
+    "TFUNCTION": {
+        "summary": "A container for tfunction commands",
+        "since": "2.0.0",
+        "group": "scripting",
+        "complexity": "Depends on subcommand.",
+        "subcommands": {
+            "TFUNCTION|LOAD": {
+                "summary": "Load a new library to Triggers and Functions",
+                "since": "2.0.0",
+                "group": "scripting",
+                "complexity": "O(1)",
+                "arguments": [
+                    {
+                        "name": "replace",
+                        "pure-token": "replace",
+                        "display_text": "replace",
+                        "token": "REPLACE",
+                        "optional": true
+                    },
+                    {
+                        "name": "config",
+                        "type": "string",
+                        "display_text": "config",
+                        "token": "CONFIG",
+                        "optional": true
+                    },
+                    {
+                        "name": "library-code",
+                        "type": "string",
+                        "display_text": "library code"
+                    }
+                ]
+            },
+            "TFUNCTION|DELETE": {
+                "summary": "Delete a library from Triggers and Functions by name",
+                "since": "2.0.0",
+                "group": "scripting",
+                "complexity": "O(1)",
+                "arguments": [
+                    {
+                        "name": "library-name",
+                        "type": "string",
+                        "display_text": "library name"
+                    }
+                ]
+            },
+            "TFUNCTION|LIST": {
+                "summary": "List all libraries loaded into Triggers and Functions",
+                "since": "2.0.0",
+                "group": "scripting",
+                "complexity": "O(N) where N is the number of libraries loaded into Triggers and Functions",
+                "arguments": [
+                    {
+                        "name": "library-name",
+                        "type": "string",
+                        "display_text": "library name",
+                        "token": "LIBRARYNAME",
+                        "optional": true
+                    },
+                    {
+                        "name": "withcode",
+                        "type": "pure-token",
+                        "display_text": "withcode",
+                        "token": "WITHCODE",
+                        "optional": true
+                    },
+                    {
+                        "name": "verbose",
```

Comment:
Can we somehow specify that this can be repeated ?

Reply:
do we have the same behaviour on another command? I couldn't find a pure-token type that had multiple in the Redis.